assets/sass/components/global/_googlesitekit-entity-header.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Entity Header Banner styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-entity-header {
+		background-color: map-get($wp-colors, "gray-2");
+		color: $c-cape-cod;
+		display: flex;
+		justify-content: space-between;
+		padding: 16px;
+		-webkit-position: sticky;
+		position: sticky;
+		top: 132px;
+		z-index: 9; // should be under _googlesitekit-dashboard-navigation
+
+		body.admin-bar & {
+
+			@media ( min-width: $width-tablet + 1 + px ) {
+				top: 194px;
+			}
+
+			@media ( min-width: $bp-wpAdminBarTablet ) {
+				top: 180px;
+			}
+
+			@media ( min-width: $bp-desktop ) {
+				top: 196px;
+			}
+		}
+
+		@media (min-width: $bp-tablet) {
+			padding: 16px 34px;
+		}
+	}
+
+	.googlesitekit-entity-header--has-scrolled {
+		box-shadow: 0 3px 3px rgb(0 0 0 / 10%);
+		transition: box-shadow 0.3s;
+	}
+
+	.googlesitekit-entity-header__back {
+		font-size: 12px;
+		font-weight: $fw-primary-normal;
+		height: auto;
+		letter-spacing: 0.3px;
+		line-height: 1;
+		margin: 4px 0 0;
+		min-width: auto;
+		text-transform: none;
+
+		@media (min-width: $bp-tablet) {
+			font-size: 16px;
+			margin: 4px 0;
+		}
+
+		&::before,
+		&::after {
+			display: none;
+		}
+
+		&.mdc-button:not(:disabled) {
+			color: $c-cape-cod;
+		}
+
+		.mdc-button__label {
+			line-height: 15px;
+			margin: 0 0 0 8px;
+			text-align: left;
+		}
+	}
+
+	.googlesitekit-entity-header__details {
+		overflow: hidden;
+		padding: 0 0 0 16px;
+		text-align: right;
+		white-space: nowrap;
+
+		p {
+			font-family: $f-secondary;
+			font-size: 11px;
+			line-height: 14px;
+			margin: 4px 0 -4px 0;
+			overflow: hidden;
+			text-overflow: ellipsis;
+
+			@media (min-width: $bp-tablet) {
+				font-size: 14px;
+				line-height: 20px;
+				margin: 0;
+			}
+		}
+
+		a {
```

Comment:
Can we add text overflow for the link as well to make it look better on mobile devices? Currently if the link is long, I don't see an icon at the end and the last part of the link.

![image](https://user-images.githubusercontent.com/873049/145825803-249ca86f-fc3f-467d-9438-d5257a44a985.png)

I think ideally it would be better if we "collapse" the central part of the URL to be something like `http://mysite.com/.../page-slug.html` instead of `http://mysite.com/categoryA/...`. 

cc @felixarntz

Reply:
Also the link looks too small and too thin on bigger screens. Maybe we should make it a bit bigger and to use the same styles as we use for source links?

![image](https://user-images.githubusercontent.com/873049/145827582-77eeca09-b3e4-4fbf-a71f-0ecb05600d20.png)

Reply:
@eugene-manuilov @jimmymadon Agreed, can we add an ellipsis to the URL instead of actually cutting it off because of overflow issues? I also agree that it'd be a better experience putting that ellipsis somewhere in between, to try to keep the last segment visible. Let's do that, unless that would require a significantly more complex approach.

cc @marrrmarrr

Reply:
@eugene-manuilov @felixarntz I've added the ellipsis in between the `entityURL` string by splitting the strings in JSX and then using CSS to add the ellipsis. The current solution always adds the ellipsis 10 characters before the end of the URL. This way, most of the initial string is visible (which usually contains the domain name, date folders, categories etc) as well as the last 10 chars. 
<img width="1453" alt="Screenshot 2021-12-14 at 14 09 30" src="https://user-images.githubusercontent.com/1721319/146014244-aec59f1f-7248-4b61-a07e-20b718e9533d.png">

This happens on all viewports for very long URLs - including mobile viewports. Since CSS takes care of the overflow now, do we still need a maximum character limit?

On a side note, I do agree with @eugene-manuilov that the font-size of the URL is tiny on hi-res (qHD+) large screens. So far I've just followed the Figma designs but let me know if I should increase this.

Reply:
@felixarntz Eugene rightly pointed out that the ending part of the URL is what is more useful for users to view as the initial part of the URLs in a site are all going to be similar. So do we have any strong preferences as to where the ellipsis should be inserted? Should it be inserted right after the origin/hostname, clipping the initial sections of the pathname?

Reply:
@jimmymadon I think it would be best to try to specifically keep the last path segment intact.

We could either put the ellipsis right before that, or maybe right after the initial slash after the domain.